html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Html Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <div class="coding-form">
```

Comment:
I think we are missing some important html element here
HINT: this exercise is `html-forms`